src/libproxychains.c

```diff
@@ -38,6 +38,12 @@
 #include "core.h"
 #include "common.h"
 
+#ifdef IS_SOLARIS
+#ifndef s6_addr32
+#define s6_addr32 _S6_un._S6_u32
```

Comment:
NOPE

i did the configure check the way it is done to not have to clutter the code with ugly ifdefs
configure sets the macro to the right value for the platform its compiled too.

Reply:
Sorry, I forgot to remove this in commit e20e0e8